centos7/Dockerfile.pgo-installer.centos7

```diff
@@ -0,0 +1,36 @@
+ARG PREFIX
+ARG BASEVER
+FROM ${PREFIX}/pgo-base:centos7-${BASEVER}
+
+LABEL name="pgo-installer" \
+    summary="Crunchy PostgreSQL Operator - Installer" \
+    description="Crunchy PostgreSQL Operator - Installer"
+
+
+# Kubernetes repository
+RUN echo $'[kubernetes] \n\
+name=Kubernetes \n\
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64 \n\
+enabled=1 \n\
+gpgcheck=1 \n\
+repo_gpgcheck=1 \n\
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg' > /etc/yum.repos.d/kubernetes.repo
+
+RUN yum -y install epel-release
+
+# which is used in the ansible roles to verify
+# used to install ssk-keyen and gen ssh keys
+RUN yum -y install \
+    --setopt=skip_missing_names_on_install=False \
+    kubectl \
+    ansible \ 
+    which
```

Comment:
Can these be combined into one `RUN` statement? I think potentially so can the other one too